User/UserContext/doc/spec.md

```diff
@@ -0,0 +1,72 @@
+# UserContext
+
+## Description
+
+This data model describe the Context of a User. No personal data is encoded in
+the model. The actual User data are stored in a different end point, as identified
+by the `refUser` property.
+
+## Data Model
+
+A JSON Schema corresponding to this data model can be found [here](../schema.json).
+
++ `id` : Unique identifier.
+
++ `type` : Entity type. It must be equal to `UserContext`.
+
++ `dateModified` : Last update timestamp of this entity.
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Optional
+
++ `dateCreated` : Entity's creation timestamp.
+    + Attribute type: [DateTime](https://schema.org/DateTime)
+    + Optional    
+
++ `refUser` : reference to the (anonymised) User to which this UserContext is associated.
+    + Attribute type: [https://schema.org/URL](https://schema.org/URL)
+    + Normative References: [https://tools.ietf.org/html/rfc3986](https://tools.ietf.org/html/rfc3986)
+    + Mandatory
+
++ `location` : Current location of the User represented by a GeoJSON geometry.
+    + Attribute type: `geo:json`.
+    + Normative References: [https://tools.ietf.org/html/rfc7946](https://tools.ietf.org/html/rfc7946)
+    + Mandatory if `address` is not defined.
+
++ `address` : Current civic address of the User
+    + Normative References: [https://schema.org/address](https://schema.org/address)
+    + Mandatory if `location` is not present.
+
++ `refDevice` : An object representing the current device user by the User. See [Device](../../Device/Device/doc/spec.md) definition.
```

Comment:
I would rename this to `refUserDevice` to distinguish it from other data models where `refDevice` is used to reference the device that provided the data